Automattic/wp-calypso
# Stats: Add year navigation to annual highlights

#### Proposed Changes
Split from #71159.

* enhancing annual highlights by adding arrows to navigate between years

#### Testing Instructions



* navigate to the `Insights` section of Stats
* verify that there are arrows present on the right side of `2022 in review` and clicking them changes visible year in review (annual stat cards below should also change)
* make sure that you can't navigate to the years not available on `View all annual insights` page - years earlier to the first data point or the future
* check that you can navigate to the arrows using a keyboard
* `Followers` count should only be visible for the current year. Past years should show an empty value represented by a hyphen.

Before:
![Screenshot 2022-12-14 at 1 52 25 PM](https://user-images.githubusercontent.com/112354940/208558128-fa1454d8-951a-4a10-839b-ced33ef06b7b.png)

After:
![Screenshot 2022-12-20 at 2 10 59 PM](https://user-images.githubusercontent.com/112354940/208558073-0cb42877-456c-4d95-810f-c0019aef52ed.png)


#### Pre-merge Checklist





Related to #70206
